scripts/build_subtargets.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
```

Comment:
Keen to check consensus/views on separate script vs just updating `build.sh` and having the one script. What were your thoughts on this @ArkaSaha30?

My initial reaction would be just to add this logic into `build.sh` so only one script is involved but keen to hear what others think and happy to go with overall consensus.

Reply:
> My initial reaction would be just to add this logic into build.sh so only one script is involved

I agree on this, I was wondering the same but thought of keeping the `build.sh` untouched since it is an alpha/beta level change 🤔